tests/converter/test_code_generation.py

```diff
@@ -298,7 +335,7 @@ def codegen_cpu(self,
                     from_npy=False,
                     need_arm_compiler=False,
                     cache_dma=False,
-                    use_avx=False,
+                    use_fpga=False,
```

Comment:
Using boolean for this flag is good. But in test cases, this flag is defined as a string.

Reply:
Maybe boolean might better as you say.
But this time, can I use string type instead?
Many all other part uses `use_fpga` as string because `use_fpga` will be argument of Makefile like `make build ARCH=aarch64 USE_FPGA=enable TYPE=exuecutable`.
If we use boolean, we need to convert it from boolean to string in the middle.

Reply:
OK! No problem.